src/components/filter/filter-amount/FilterAmount.tsx

```diff
@@ -38,53 +39,53 @@ export const FilterAmount = ({
   const [filterType, setFilterType] = useState<FilterAmountType>(
     filter.amountFilterType,
   );
-  const [editingAmount1, setEditingAmount1] = useState<number | null>(
+  const [editingAmountMin, setEditingAmountMin] = useState<number | null>(
     filter.amountTotalMin,
   );
-  const [editingAmount2, setEditingAmount2] = useState<number | null>(
+  const [editingAmountMax, setEditingAmountMax] = useState<number | null>(
     filter.amountTotalMax,
   );
   const [menuOpen, setMenuOpen] = useState(false);
 
   const dispatchAmounts = useCallback(
-    (values: { amount1: number | null; amount2: number | null }) => {
+    (values: { amountMax: number | null; amountMin: number | null }) => {
       dispatch({
         name: 'amountTotalMin',
         type: 'change',
-        value: values.amount1,
+        value: values.amountMin,
       });
       dispatch({
         name: 'amountTotalMax',
         type: 'change',
-        value: values.amount2,
+        value: values.amountMax,
       });
     },
     [dispatch],
   );
 
   const handleApplyFilterText: FilterApplyCallback = useCallback(
     setFilterText => {
-      const amount1String = `${editingAmount1?.toFixed(2) || 0}` || '';
-      const amount2String = `${editingAmount2?.toFixed(2) || 0}` || '';
+      const amountMinString = `${editingAmountMin?.toFixed(2) || 0}` || '';
+      const amountMaxString = `${editingAmountMax?.toFixed(2) || 0}` || '';
```

Comment:
I think it would never fall back to ''. Because you are trying a fallback with a string interpolation, it would always fall back to '0' if editingAmount is `null`.